sql/core/src/main/scala/org/apache/spark/sql/internal/SessionState.scala

```diff
@@ -127,8 +125,6 @@ private[sql] object SessionState {
 /**
  * Concrete implementation of a [[BaseSessionStateBuilder]].
  */
-@Experimental
-@Unstable
```

Comment:
For `SessionStateBuilder`, we may want to keep `@Unstable` like `HiveSessionStateBuilder.scala` is `@Unstable` in this PR.

cc @gatorsmile , @cloud-fan , @hvanhovell

Reply:
Yeah if in doubt let's just leave it in, but, yeah I also don't know if there's doubt or what as this hasn't changed in a long while.